portfolio/src/main/webapp/index.html

```diff
@@ -11,78 +11,81 @@
       <h1>a page about me</h1>
       <p>this is my portfolio, a little more about B</p>
     </div>
-    <br>
 
-    <div id="about-sidebar">
-        <p>about me</p>
-        <br>
-        <p>NYU | STEP Intern | art</p>
-        <br>
-        <a href="https://www.linkedin.com/in/bianca-macias-344a26178">LinkedIn</a>
-        <br>
-        <a href="https://github.com/biancamacias/">Github</a>
-        <br>
-
-        <img src="/images/me1.jpg" alt="picture of me" width="100" height="100">
-        <br>
-        <img src="/images/me2.jpg" alt="picture of princess and me" width="100" height="100">
-        <br>
-        <img src="/images/me3.jpg" alt="picture of princess and me again" width="100" height="100">
-        <br>
-        <img src="/images/me4.jpg" alt="picture of me again" width="100" height="100">
-        <br>
-    </div>
-
-    <div id="content-sidebar">
-        <div class="blog-post">
-            <p>Welcome to my page! I'm a rising sophomore at New York University. 
-                If I'm not studying for my computer science degree, I'm usually in a sunny room 
-                sketching and painting or writing. I only started coding 7 months ago, and I am most 
-                proficient in Python, and it's my favorite language so far because I don't worry about 
-                semi-colons or private variables or declaring the datatypes of different data structures. 
-                I'm an artist at heart, so computer science does not come easy to me, but I love the thrill of 
-                discovering new ways to display ideas to other people, which is what drew me to pursue a career in 
-                this field.</p>
+    <div id="body-content">
+        <div id="about" class="sidebar">
+            <div id="links" class="box">
+                <p id="about-title">a crash course in me</p>
+                <br>
+                <p id="about-content">NYU | STEP Intern | art</p>
+                <br>
+                <a href="https://www.linkedin.com/in/bianca-macias-344a26178">LinkedIn</a>
```

Comment:
nit - consider adding `target="_blank"` to have this link and the github link open in a new tab.